docs/source/tutorials/artifact.rst

```diff
@@ -0,0 +1,258 @@
+========
+Artifact
+========
+
+A data artifact is a bundle of input data associated with a particular
+model. It is typically stored as an ``hdf`` file on disk with very particular
+formatting. This file is then used by the :mod:`vivarium` simulations to fill
+in all the relevant parameter data.
+
+It is frequently useful to be able to view or modify this data outside the
+simulation.  The :class:`vivarium.Artifact` provides a high level interface to
+do just that. In this tutorial we'll go through how to view, delete, and write
+data to an artifact using the tools provided by the
+:class:`~vivarium.Artifact`. You'll access data in the artifact through keys,
+mirroring the underlying hdf storage of artifacts.
+
+.. contents::
+   :depth: 1
+   :local:
+   :backlinks: none
+
+
+Creating an artifact
+---------------------
+
+To view an existing hdf file via the :class:`~vivarium.Artifact` tools, we'll
+create a new artifact. We can print the resulting artifact to view the tree
+structure of the keys in our artifact. We'll use our test artifact to
+illustrate:
+
+.. code-block:: python
+
+    from vivarium import Artifact
+
+    art = Artifact('test_artifact.hdf')
+    print(art)
+
+::
+
+    Artifact containing the following keys:
+    metadata
+            keyspace
+            locations
+            versions
+    population
+            age_bins
+            structure
+            theoretical_minimum_risk_life_expectancy
+
+Now we have an :class:`~vivarium.Artifact` object, which we can use to interact
+with the data stored in the hdf file with which we created it.
+
+
+Filter Terms
++++++++++++++
+
+The data stored in artifacts may be large, potentially on the order of millions
+of rows for a single dataset, and loading a full dataset requires time and
+memory, both of which may be limited. If you are only interested in certain
+subsets of the data, either rows or columns, you may want to read only the
```

Comment:
I don't think you can filter to select columns besides draw columns via the artifact - you can via the raw hdf but the Artifact only exposes a single set of filter terms, splits off the term relevant to draws and then passes that and the remaining terms onto the underlying hdf separately. I don't know if it's worth making that distinction here/if it really matters, but the way this sentence is written, it does make it sound like you could select any set of columns